configure

```diff
@@ -2823,18 +2822,14 @@ case $_host_os in
 			_optimization_level=-O0
 			define_in_config_if_yes "$_debug_build" 'DEBUG_BUILD'
 		fi
-		# dynamic builds leave out resources to save binary size
-		# and detection features *must* be static
+		# dynamic builds will leave out resources to save binary size
 		if test "$_dynamic_modules" = yes ; then
 			_builtin_resources=no
+			_detection_features_static=no
 			_plugins_default=dynamic
-		else
```

Comment:
So what will happen now when the user selects to build with static plugins?

Reply:
if no
--enable-plugins
switch is used, configure will instead set
_plugins_default=static
_static_build=yes
by default